scripts/train.py

```diff
@@ -179,7 +204,7 @@ def main(cfg):
     # Train the model.
     ######################################################################
 
-    trainer.fit(model, datamodule=datamodule)
+    trainer.fit(model, train_loader, [val_loader, unseen_loader])
```

Comment:
add the train_loader to the list of validation loaders, because we want "clean" metrics on the train set (not because of data augmentations, but because during train mode `model.eval()` is not set, so weird stuff happens with batch norm and dropout

Reply:
Here do you mean doing something like this: trainer.fit(model, train_loader, [train_loader, val_loader, unseen_loader]) ?

Reply:
Yep, exactly that.